src/optimizations/optimizeMeshes.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Cognite AS
+
+import Sector from '../Sector';
+import { MergedMesh, MergedMeshMappings } from '../geometry/MergedMeshGroup';
+import * as THREE from 'three';
+import { InstancedMeshCollection } from '../geometry/InstancedMeshGroup';
+import SceneStats from '../SceneStats';
+import { TreeIndexNodeIdMap } from '../parsers/parseUtils';
+
+const globalMatrix = new THREE.Matrix4();
+
+const TRIANGLE_COUNT_LIMIT = 10000;
+
+function countMappingsToMerge(collections: InstancedMeshCollection[]) {
+  let numMappings = 0;
+  collections.forEach(collection => {
+    const triangleCount = collection.mappings.count * collection.triangleCount;
+    if (triangleCount < TRIANGLE_COUNT_LIMIT) {
+      numMappings += collection.mappings.count;
+    }
+  });
+  return numMappings;
+}
+
+export default function optimizeMeshes(
```

Comment:
I still think this is a bad function name unless it does a series of optimizations that do have good names. `mergeInstancedMeshes` is telling you more about what's happening.